sw/device/silicon_creator/lib/cert/dice_cwt.c

```diff
@@ -5,29 +5,63 @@
 #include <stdint.h>
 
 #include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/runtime/print.h"
+#include "sw/device/silicon_creator/lib/base/util.h"
 #include "sw/device/silicon_creator/lib/cert/cbor.h"
 #include "sw/device/silicon_creator/lib/cert/cert.h"
 #include "sw/device/silicon_creator/lib/cert/dice.h"
 #include "sw/device/silicon_creator/lib/drivers/hmac.h"
 #include "sw/device/silicon_creator/lib/drivers/keymgr.h"
 #include "sw/device/silicon_creator/lib/drivers/otp.h"
 #include "sw/device/silicon_creator/lib/error.h"
+#include "sw/device/silicon_creator/lib/otbn_boot_services.h"
 #include "sw/device/silicon_creator/lib/sigverify/ecdsa_p256_key.h"
 #include "sw/device/silicon_creator/manuf/lib/flash_info_fields.h"
 
 #include "otp_ctrl_regs.h"  // Generated.
 
-// Keys
-const int64_t kCoseKeyKtyLabel = 1;
-const int64_t kCoseKeyAlgLabel = 3;
-const int64_t kCoseEc2CrvLabel = -1;
-const int64_t kCoseEc2XLabel = -2;
-const int64_t kCoseEc2YLabel = -3;
+// Labels
+#define CWT_COSE_KEY_LABEL_TYPE (1)
```

Comment:
Can most of these be replaced by enum values? https://opentitan.org/book/doc/contributing/style_guides/c_cpp_coding_style.html#constants-and-preprocessor-macros

Reply:
Sure. These values will be defined in the cwt-template.